src/arrayMethodSort.js

```diff
@@ -4,9 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareString) {
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const previous = this[i - 1];
+        const current = this[i];
+
+        if (compareFunction(previous, current) > 0) {
+          this[i - 1] = current;
+          this[i] = previous;
+          count++;
+        }
+      }
+    } while (count);
+
+    return this;
   };
 }
 
+const compareString = function compareItems(a, b) {
```

Comment:
You can just create your function, why do you assign it to the new variable?

```suggestion
function compareString(a, b) {
```